### PR TITLE
fix(code-snippet): update button active token

### PR DESCRIPTION
**Pull request by SimonFinney**
Experimental code snippet button active state references hard-coded `$ibm-colors__*` values instead of a token role.

#### Changelog

**Changed**

- Update code snippet button active token

### CODE REVIEW

File: src/components/code-snippet/_code-snippet.scss [hunk 1]
```diff
@@ -299,6 +299,18 @@
 }
 
 @mixin snippet--x {
+  .#{$prefix} {
```
Comment (joshblack) [inline src/components/code-snippet/_code-snippet.scss]: Small nit, for new styles we're trying to keep selectors un-nested for discoverability later on in the code lifecycle. Would you mind un-nesting this block to help follow this?

**Review by joshblack — CHANGES_REQUESTED** [review]

File: src/components/code-snippet/_code-snippet.scss (hunk 1 above)
Comment (SimonFinney) [reply src/components/code-snippet/_code-snippet.scss]: Absolutely - I thought that might come up and had a feeling that was the case 😅

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @SimonFinney for the update!

**Review by tw15egan — APPROVED** [review]
Looks good to me when merge conflicts are resolved 👍

**Comment by asudoh** [comment]
@joshblack - Did you have any further comments? Thanks!

**Comment by asudoh** [comment]
Let's go forward as we have two reviewers sign-off. Anybody don't hesitate to speak up if you have any concerns - Thanks!

**Pull request merged**